diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7c82b3a..48474ec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -27,6 +27,8 @@ project(absl LANGUAGES CXX VERSION 20240722)
 set(ABSL_SOVERSION "2407.0.0")
 include(CTest)
 
+add_compile_options(-pthread)
+
 # Output directory is correct by default for most build setups. However, when
 # building Abseil as a DLL, it is important to have the DLL in the same
 # directory as the executable using it. Thus, we put all executables in a single
